spacepy/coordinates.py

```diff
@@ -392,6 +399,71 @@ def append(self, other):
         newobj = Coords(data, dtype=self.dtype, carsph=self.carsph)
         return newobj
 
+    # -----------------------------------------------
+    def to_skycoord(self):
+        '''
+        Create an Astropy SkyCoord instance based on this instance
+
+        Returns
+        =======
+        out : `astropy.coordinates.SkyCoord`
+            This coordinate as an Astropy SkyCoord
+
+        Notes
+        =====
+        This method requires Astropy to be installed.
+
+        This method uses the GEO coordinate frame as the common frame between the two libraries.
+        '''
+        if self.ticks is None:
+            raise ValueError("This method requires the attribute `ticks` to be specified.")
+
+        try:
+            import astropy.coordinates
+            import astropy.time
+        except ImportError as e:
+            raise e.__class__("This method requires Astropy to be installed.")
+
+        coord = self.convert('GEO', 'car')  # GEO is Astropy ITRS
+        data = coord.data * self.Re  # convert to meters
```

Comment:
This assumes the input coordinates are in Re, but they can be (e.g.) km. We're not very good about handling this elsewhere--for instance, irbempy calls assume they're Re regardless of the units attribute--but it's probably worth doing something.

The fairly fancy approach would be to support Re, km, m and throw an error if they 1) aren't the same unit in all dimensions 2) not one of the above. I don't want to overburden this PR with fanciness. @drsteve , thoughts?

Reply:
My thought is that that's out of scope here. The documentation isn't as clear as it could be, but all Cartesian coordinate systems are assumed to be in Re, with spherical systems in degrees and Re. The only one that's different is geodetic, which is assumed/required to be in degrees and km. There's checking for that in #296, when we finally get to working through that (or breaking it for parts). The units are currently there to tell the user, not to let the user switch...

So this PR does the right thing for a correctly specified `Coords` per documentation. I'm happy as is.